Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocList.css

```diff
@@ -102,6 +102,47 @@ div.bocListNavigator a[href]:active img
   filter: progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
 }
 
+div.bocList div.bocListTableBlock tr.bocListValidationRow.even
```

Comment:
I didn't review this in detail. the NovaGray is simpler and with a lot more css duplication.